### ISSUE TITLE
patched plugin.xml file not being included in jar

### ISSUE THREAD

**Issue by codingismy11to7** [JetBrains/sbt-idea-plugin#58]
I just updated from plugin version 2.3.2 to version 3.6.3, and implemented the new `patchPluginXml` settings. After running `packageArtifactZip`, I checked to see if everything was correct, but the plugin.xml included in the jar was the original, unpatched file.

I did check that the file output to `target/scala-2.12/classes/META-INF/plugin.xml` DID get the patched values correctly, it just seems that the jar after bundling is using the original.

```patch
--- scala-2.12/classes/META-INF/plugin.xml	2020-04-13 14:47:04.000000000 -0400
+++ plugin-jar-unpacked/lib/META-INF/plugin.xml	2020-04-13 14:46:44.000000000 -0400
@@ -1,10 +1,10 @@
 <idea-plugin url="http://www.jetbrains.com/idea">
     <id>com.codmettle.plugin</id>
     <name>Plugin Name</name>
-    <version>1.5</version>
+    <version>1.4</version>
     <vendor url="http://www.codemettle.com">CodeMettle</vendor>
     <description>IntelliJ IDEA plugin to support Product Development</description>
-    <idea-version since-build="201.6668.121"/>
+    <idea-version since-build="183.0"/>
     <depends>org.intellij.scala</depends>
     <application-components>
         <!-- TODO put your components here -->
```

**Comment by codingismy11to7** [comment]
thanks!

**Comment by mutcianm** [comment]
plz test and report if it works

**Comment by codingismy11to7** [comment]
I'll try to but it probably won't be until next week, and I'll have to figure out how to do that (hopefully as easy as a `publishLocal` and updating the build to use a snapshot version for the plugin

**Comment by mutcianm** [comment]
All the fixes are already in 3.7.0
I've also fixed quite a few other annoying issues, that you've might came across: #63 #62 #61 #59 tl;dr:
- jetbrains runtime is used in generated run configurations in IDEA(fixes various OSX issues)
- no need to manually call sbt tasks after importing project anymore(or using sbt-shell) to generate artifacts and run configurations in IDEA
- JUnit template is generated on import
